From b664d445af087515b1d444380b4360c1c0216459 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Thu, 8 May 2014 18:03:44 -0400
Subject: [misc] mei: client.h fix checkpatch errors

Message-id: <1399572224-7552-1-git-send-email-prarit@redhat.com>
Patchwork-id: 79576
O-Subject: [RHEL7.1 PATCH BZ 1088953 71/76] mei: client.h fix checkpatch errors [v2]
Bugzilla: 1088953
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1088953

commit 1adc16743313688ba69d19d9a445c60affc66ef9
Author: Tomas Winkler <tomas.winkler@intel.com>
Date:   Mon Mar 17 23:57:23 2014 +0200

    mei: client.h fix checkpatch errors

    Fix checkpatch error
    return is not a function, parentheses are not required

    Signed-off-by: Tomas Winkler <tomas.winkler@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

[v2]: previous patch changes required an update to this patch

Cc: Dean Nelson <dnelson@redhat.com>
Cc: Stefan Assmann <sassmann@redhat.com>
Cc: Tony Camuso <tcamuso@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/misc/mei/client.h b/drivers/misc/mei/client.h
index 8139393..96d5de0 100644
--- a/drivers/misc/mei/client.h
+++ b/drivers/misc/mei/client.h
@@ -68,15 +68,15 @@ int mei_cl_flow_ctrl_reduce(struct mei_cl *cl);
  */
 static inline bool mei_cl_is_connected(struct mei_cl *cl)
 {
- return (cl->dev &&
+ return  cl->dev &&
   cl->dev->dev_state == MEI_DEV_ENABLED &&
-  cl->state == MEI_FILE_CONNECTED);
+  cl->state == MEI_FILE_CONNECTED;
 }
 static inline bool mei_cl_is_transitioning(struct mei_cl *cl)
 {
- return (MEI_FILE_INITIALIZING == cl->state ||
+ return  MEI_FILE_INITIALIZING == cl->state ||
   MEI_FILE_DISCONNECTED == cl->state ||
-  MEI_FILE_DISCONNECTING == cl->state);
+  MEI_FILE_DISCONNECTING == cl->state;
 }
 
 bool mei_cl_is_other_connecting(struct mei_cl *cl);
-- 
1.7.1